v0.5/classification_and_detection/python/main.py

```diff
@@ -79,6 +82,15 @@
         "max-batchsize": 32,
     },
 
+    # vgg19
```

Comment:
Sorry, VGG19 is not part of MLPerf Inference v0.5. Adding it in the official code would be confusing for users.